docs/01_setup.md

```diff
@@ -64,17 +64,21 @@ You can also configure your own schemas here if you wish to, or deploy the EAS c
 
 As a coordinator you need to deploy a MACI instance and poll.
 
-### Install Maci
+### Install MACI
 
 You can read about the [MACI requirements here](https://maci.pse.dev/docs/installation). To install MACI run the following commands:
 
 ```bash
 git clone https://github.com/privacy-scaling-explorations/maci.git && \
 cd maci && \
+git checkout v1.2.2 && \
 pnpm i && \
 pnpm run build
 ```
 
+> [!IMPORTANT]
+> It's important to use version 1.2.2 of MACI, as this version has ceremony keys and has been audited.
```

Comment:
please can you change the part after the comma to: "as this version's circuit are audited and have zKeys which have undergone a trusted setup".  The contracts have some small changes (removal of subsidy and Mode for NonQv vs Qv) since the audit, so worth being accurate with wording here.